modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -0,0 +1,109 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-command-reference_{context}"]
+= Pipelines as Code command reference 
+
+[role="_abstract"]
+The `tkn-pac` CLI tool offers the following capabilities:
+
+* Bootstrap Pipelines as Code installation and configuration
+* Create a new Pipelines as Code repository
+* List all Pipelines as Code repositories
+* Describe a Pipelines as Code repository and the associated runs
+* Generate a simple pipeline run to get started
+* Resolve a pipeline run, as if it was executed by Pipelines as Code on service
```

Comment:
```suggestion
* Resolve a pipeline run as if it was executed by Pipelines as Code on service
```
- omit comma
- This phrase doesn't make sense to me `by Pipelines as Code on service`